internal/server/proto/server.proto

```diff
@@ -2828,6 +2831,27 @@ message GetStatusReportRequest {
   // a report by deployment or release operation id
 }
 
+message ExpediteStatusReportRequest {
+  Ref.Operation ref = 1;
+
+  // Target is used to determine what a status report should be run against
+  oneof target {
+    // the deployment id that this status report was generated on
+    Deployment deployment = 2;
+
+    // the release id that this status report was generated on
+    Release release = 3;
```

Comment:
I was expecting these to be `Ref.Deployment` and `Ref.Release` (rather than embedding whole `Deployment` and `Release` messages) but of course we don’t actually have `Ref.Deployment` and `Ref.Release` in the schema, just `Ref.Operation` which doesn’t tell you what type of operation is being referenced.

What do you think is the best course of action? Add `Ref.Deployment` and `Ref.Release`?

Reply:
We could add those, that isn't so bad. Then the UI passes along the name inside one of the Refs rather than the entire message. Will push that commit now.

Reply:
Or it could potentially be something like this:

```proto
message ExpediteStatusReportRequest {
  // Target is used to determine what a status report should be run against
  oneof target {
    Ref.Operation deployment = 1;
    Ref.Operation release = 2;
  }
}
```

Reply:
Haha I was about to suggest that! We can get the deployment or release from the Ref_Operation, so yep, that's what we need 👍🏻